### DISCUSSION TITLE
Please add FreeBSD to the list of operating systems where Halide can run

### DISCUSSION THREAD

**Discussion by yurivict** [halide/Halide#4956] [General]
On your website https://halide-lang.org/
* Operating systems: Linux, Windows, macOS, Android, iOS, Qualcomm QuRT

https://www.freshports.org/lang/halide/

**Comment by alexreinking** [comment]
We don't have first-party support FreeBSD, unlike the others in that list. We could consider linking to this and other (?) ports in a separate section of the webpage.
